olp-cpp-sdk-core/include/olp/core/cache/DefaultCache.h

```diff
@@ -178,6 +178,27 @@ class CORE_API DefaultCache : public KeyValueCache {
    */
   bool Contains(const std::string& key) const override;
 
+  /**
+   * @brief Protect a list of keys from eviction.
+   * Keys or prefixes to protect single keys or entire catalogs, layers,
+   * version.
+   *
+   * @param keys list of keys and prefixes.
+   *
+   * @return True if the keys added to protected list; false otherwise.
+   */
```

Comment:
```suggestion
  /**
   * @brief Checks whether keys are protected from eviction.
   *
   * You can use keys or prefixes to protect single keys or entire catalogs, layers,
   * version.
   *
   * @param keys The list of keys and prefixes.
   *
   * @return True if the keys are added to the protected list; false otherwise.
   */
```
Not sure if I got the brief part right.